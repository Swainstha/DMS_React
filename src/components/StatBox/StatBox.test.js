import testSnapFunction from '../../util/testSnapFunction';
import StatBox from './index'
import React from 'react';
const dummy_data= {
types:"type 1"
}

testSnapFunction("<StatBox />", "Snapshot test for StatBox ",<StatBox {...dummy_data}><div>dsjfM</div> </StatBox> )