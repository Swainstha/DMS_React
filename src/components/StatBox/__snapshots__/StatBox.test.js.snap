// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StatBox /> Snapshot test for StatBox  1`] = `ShallowWrapper {}`;
